src/stories/3-fields/date-input/date-input.stories.tsx

```diff
@@ -0,0 +1,137 @@
+import { ArgsTable, Description, Heading, PRIMARY_STORY, Stories, Title } from "@storybook/addon-docs";
```

Comment:
Actually we might want to use full mdx in future as we can customise the styling and layout. But it is fine, we can stick to this for now